tests/bam/bam_pb.robot

```diff
@@ -1291,6 +1291,50 @@ BA_DISABLED
 
     [Teardown]    Ctn Stop Engine Broker And Save Logs    ${True}
 
+BA_SERVICE_PNAME_AFTER_RELOAD
+    [Documentation]    Given a BA with her service, parent_name is not erased after cbd reload.
```

Comment:
[Documentation]    Scenario: Verify that the parent_name of a BA service is not erased after a broker reload
...    Given a BA "test" of type "worst" with its service "host_16:service_302"
...    When I start broker and engine
...    Then the BA service "test" should have a status of 0 within 30 seconds
...    When I reload the broker
...    Then the database should still contain a BA service with name "test" and parent_name "_Module_BAM_1"